_docs/useful-links.md

```diff
@@ -29,42 +29,36 @@ The following is a list of links for data providers and web applications develop
 
   The IOOS Compliance Checker is a python based tool for data providers to check for completeness and community standard compliance of local or remote netCDF files against metadata standards. This web based tool can be used to check your GliderDAC netCDF files for compliance. Be sure to select **GliderDAC** in the list of available tests to run.
 
- - [**FTP script**](https://github.com/ioos/glider-dac/blob/master/util/ncFtp2ngdac.pl)
```

Comment:
@benjwadams can you give your :+1: or :-1:  here? As far as I remember this is deprecated, right?